README.md

```diff
@@ -108,6 +108,14 @@ let templateValues = {
   };
 ```
 
+###### Special Template Values
```

Comment:
what about a different option to separate these from templateValues ?

Reply:
Addressed in d15e665